### DISCUSSION TITLE
Listening Radio channel

### DISCUSSION THREAD

**Discussion by Trapper444** [FakeFishGames/Barotrauma#13046] [Ideas and suggestions]
The idea is for you to be able to be in 2 voice channels at the same time. Bottom channel would have a mic and the top one a headset, the idea is that you would have the bottom channel as it currently is, and the second one would be a radio channel that you could listen to, that way it could be much easier to organize people in higher player count servers, easy to divide them in for example Security and medics in channel 1, enginer and mechanics in channel 2, while they all listen to channel 0 which could be the capitan giving orders, if there is an EVA missions the members of the team would switch to channel 3 to not disturb people in the rest of the channel but still be able to listen to capitan orders

I ve done this suggestion in the discord but forgot you dont actually pay attention to the suggestions in there so here i am going to make it again in case you didn't check it out.

**Comment by bookslayer10** [comment]
I like the concept oc this idea, and I think it should be expanded further. You should be able to set any number of channels to listen to and any number of channels to speak to.

Public lobbies are currently awful when it comes to radio management. Everyone just talks on one channel because it's very risky to isolate yourself when there might be an emergency at any time. Being able to listen for emergency announcements while only talking between fellow divers on a mission would be invaluable.

This also opens up the potential for restricted radio channels. What if the Captain had his own channel that nobody else can turn off or speak into, and is used only in emergencies? What if each class had dedicated frequencies? What if there was a way to radio someone directly in an encrypted channel, either to get their attention faster than hailing them on global chat.. or for a more sinister purpose?

**Comment by rkcf** [comment]
another possible way to implement this would be placable loudspeakers that can tune into a voice channel. You could have emergency broadcasts over the loudspeaker for everyone to hear.
